tasks.py

```diff
@@ -444,7 +444,7 @@ def dev_env(ctx, architecture="amd64", name="kind", protocol=None, frr_volume_di
 
     frr_k8s_ns = "frr-k8s-system"
     if bgp_type == "frr-k8s-external":
-        run("{} apply -f https://raw.githubusercontent.com/metallb/frr-k8s/v0.0.14/config/all-in-one/frr-k8s.yaml".format(kubectl_path), echo=True)
+        run("{} apply -f https://raw.githubusercontent.com/metallb/frr-k8s/{}/config/all-in-one/frr-k8s.yaml".format(kubectl_path,get_frr_k8s_latest), echo=True)
```

Comment:
this is always going to run inv with the latest frr-k8s, which is not exactly what we want, because CI might start to fail because of sudden dependencies. We want to be deliberate in when to bump.

The way I see it, get_frr_k8s_latest should be used in this context to bump a variable used here (plus to fetch the libraries, change the charts, and the kustomize manifests). 

Also, I'd split the two steps: bump to a specific version provided by the dev OR fetch the latest version and use that.

Hope it makes sense